vendor/modules.txt

```diff
@@ -1,18 +1,8 @@
 # bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690 => bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
```

Comment:
what is the diff in this file about?

Reply:
`go mod vendor` writes more things to this file when the go.mod file indicates go1.14 is in use

Reply:
we are using 1.14 in CI etc. though right? won't this just get written back by the next person?

Reply:
it is based on the go directive in the go.mod file, not the go version used

Reply:
I tend to assume that presubmits will catch incompatibilities with CI (e.g. pull-kubernetes-verify in this case). :crossed_fingers: